doc/releases_and_migration/migration_guide_addon_v010.rst

```diff
@@ -1,101 +1,98 @@
 .. _migration_guide_addon_v010:
 
-Migration Guide for moving to Sidewalk Add-on 
-############################################
+Migration guide for moving to Sidewalk Add-on
```

Comment:
@ktaborowski should we add a release number here? The one that will be used for add-on? So it would be:

Migration guide for moving to Sidewalk Add-on (release v1.0.0)

not sure if the number is correct

Reply:
I think it's better with no version - w plan to made a v1.0.0 add on version on next stable ncs, and the instruction should be valid there as well. Also there might be more v0.X.Y tags before for testing purposes